resources/views/location/show.blade.php

```diff
@@ -39,6 +39,13 @@
                     {{ $post->body }}
                 </p>
 
+                @if($post->author_type !== App\AnonUser::class)
```

Comment:
Think about intention-revealing code and reducing brittleness. In this case, you're reaching into post for author_type and saying that that field will be a class string and this is one of the class options.

What if this said `$post->hasAnonymousAuthor()`?

Reply:
then, it would be more encapsulated. fixed 👍